### DISCUSSION TITLE
output: RT and iRT

### DISCUSSION THREAD

**Discussion by kcasctivcr** [vdemichev/DiaNN#441] [Q&A] [answered]
Hi Vadim,

I wonder what do the RT, iRT, predicted RT and predicted iRT columns in the main output stand for?
I cannot find their description on the paper nor github. I guessed that predicted RT is the theoretical retention time based on the property of the peptides, and predicted iRT is the theoretical rescaled retention time based on the retention time of the iRT peptide.
So RT would be the experimental retention time for each peptides, and iRT is the experimental iRT-rescaled retention time?

However my assumption does not seem right according to the output data.
![image](https://user-images.githubusercontent.com/65659359/179756749-8e11594e-c6d2-4e98-ab32-66e3f6bc6f19.png)

I rearranged the RT value from smallest to largest, and clearly the corresponding iRT and predicted RT value does not follow this trend, we can even see that RT values can have one-to-many relationship with iRT and predicted RT values. However the predicted iRT values seem to display better correlation with RT values, so I get pretty confused, perhaps some of my assumptions are wrong? Or is there some more detailed information about these values in the main output columns?

Thanks in advance,
Hung

**Comment by vdemichev** [comment, marked as answer]
Hi Hung,

For each run an mapping iRT (library values ) <-> RT (experimental values) is established. The 'Predicted' values reported are obtained with this mapping, e.g. Predicted.RT is the prediction of RT based on the iRT value.

Best,
Vadim

**Reply by kcasctivcr** [reply]
Hi Vadim,

I'm a still little confused. So what is Predicted.iRT? And how come Predicted.iRT correlates better with RT than iRT or Predicted.RT?

Best,
Hung

**Reply by vdemichev** [reply]
Predicted.iRT is the prediction of iRT based on the measured RT value.

**Reply by kcasctivcr** [reply]
Hi Vadim,

The reason I asked about the definition of these RT related information is that, if a peptide carries different charges (but same sequence), DIA-NN would report them in multiple rows with different quantity values, as in the example:

![image](https://user-images.githubusercontent.com/65659359/181507585-df276e5d-63af-414a-b80d-f0e184ca89d2.png)

I wonder how can I determine the quantity of this peptide?  Should I sum their "Precursor.Normalised" values? Or should I sum their PG.MaxLFQ values? What I wanted is the quantity information for peptidomic study.
Also, this is a case that the RT values only differs a little between different charges on a same peptide, if the RT values are really different, but the peptide sequence is the same, how should I deal with it?

Best,
Hung

**Reply by vdemichev** [reply]
Hi Hung,

The best is to take the maximum of precursor quantities (Precursor.Normalised or Precursor.Quantity) corresponding to the peptide.

> if the RT values are really different, but the peptide sequence is the same, how should I deal with it?
Means one of the precursor is incorrectly identified, e.g. could have been matched to a different peptidoform.

Best,
Vadim

**Reply by kcasctivcr** [reply]
Hi Vadim,

"Means one of the precursor is incorrectly identified, e.g. could have been matched to a different peptidoform."

Then how could I decide which one is the wrong one? Or should I just discard them?
Since I am working on peptidomics, this may lead to false quantification of peptides, right?

Thanks,
Hung

**Reply by kcasctivcr** [reply]
Hi Vadim,

Any thoughts about this situation?

Best,
Hung

**Reply by vdemichev** [reply]
Ourselves we don't do anything about it. Statistical FDR control ensures that error rate is controlled. Also, can discard the ID with the worse q-value.

**Reply by vdemichev** [reply]
Or can keep the IDs with better Ms1.Profile.Corr or Mass.Evidence.

**Reply by kcasctivcr** [reply]
Thanks for the responses and suggestions!

Many thanks,
Hung
